i18n/ko.json

```diff
@@ -938,9 +938,9 @@
     },
     "fact": {
         "fun": {
-            "bigThreeDominance": "재미있는 사실: 방탄소년단, 블랙핑크, 트와이스의 합산 유튜브 조회수는 {{bigThreeViews}}회 또는 {{proportion}}%를 차지 합니다.",
+            "bigThreeDominance": "재미있는 사실: 방탄소년단, 블랙핑크, 트와이스를 합친 유튜브 조회수는 {{bigThreeViews}}회, {{proportion}}%입니다!",
             "birthday": "재미있는 사실: {{name}}의 생일은 이번 달 {{formattedDate}}입니다!",
-            "companyByArtistCount": "재미있는 사실: {{company}}는 총 {{num}}명의 아티스트 (서브유닛 및 솔로 데뷔 포함)가 가장 {{ordinalNum}}명인 엔터테인먼트 회사입니다!",
+            "companyByArtistCount": "재미있는 사실: {{company}}은(는) 총 {{num}}명의 아티스트 (유닛 및 솔로 데뷔 포함)를 보유하고 있는 엔터테인먼트 회사입니다.",
```

Comment:
ordinalNum is missing

Reply:
Subtracting `{{ordinalNum}}` is intentional, as it seems more appropriate for Korean sentences.

Reply:
add, We have made corrections to this point.